### DISCUSSION TITLE
Remove settings after deactivating settings

### DISCUSSION THREAD

**Discussion by SuperFlySly** [sulu/sulu#7532] [Q&A] [answered]
Hi guys,

after I enabled some settings in a content, I deactivated these settings, but settings are still present. Do I have to remove the whole Block?

<img width="572" alt="Bildschirmfoto 2024-07-20 um 11 43 29" src="https://github.com/user-attachments/assets/f83be2a1-01da-47e6-8400-3af71736a42f">

**Comment by mamazu** [comment, marked as answer]
Sulu has made the design decision that deleted content should still be kept, so that if you choose to re-enable this feature the old values are still there and ready to be used. If you want to remove those values, then you need to remove the block, run the clean command and then recreate it.
